Dependencies/ShaderFixes/3dvision2sbsps.hlsl

```diff
@@ -38,9 +38,37 @@ void main(float4 pos : SV_Position0, out float4 result : SV_Target0)
 			x += width / 2;
 		}
 	}
+	else if (mode == 6)
+	{
+		 int side = y - (int)floor(y /2.0) * (int)2;
+    
+		 if (side == 0)
+		{
```

Comment:
Small note here-
We'll want to use the same formatting style as the surrounding code.  We don't have a strong style guide, but it's important to not inject different styles into a given block of code.
Your brace style here doesn't match the prior code, and there is value in having them match.